### ISSUE TITLE
set up example site

### ISSUE THREAD

**Issue by afeld** [18F/glossary#9]
...or at least add screenshots in the README. As a potential user, I want to see it in action!

**Comment by brittag** [comment]
Yes! It'd be wonderful to explain in the README a little more about this project - for example, is this the same code being used live at [beta.fec.gov](https://beta.fec.gov/)? Should this include a screenshot of that site? :)

**Comment by afeld** [comment]
> is this the same code being used live at beta.fec.gov?

Yep! Now stated explicitly in the README.

> Should this include a screenshot of that site?

👍
